index.html

```diff
@@ -0,0 +1,532 @@
+                                    <p class="works-tag-text">html</p>
+                                </div>
+                            </div>
+    
+                            <div class="frame43-card-works-right-block-bootstrap-tag">
+                                <div class="frame43-card-works-right-block-tag">
+                                    <p class="works-tag-text">bootstrap</p>
+                                </div>
+                            </div>
+    
+                            <div class="frame43-card-works-right-block-ruby-tag">
+                                <div class="frame43-card-works-right-block-tag">
+                                    <p class="works-tag-text">Ruby</p>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+
+                    <div class="frame43-card-works-action">
+                        <div class="frame43-card-works-action-action"></div>
+                        <button class="see-project"><p class="see-project-text">See Project</p></button>                        
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!-- About Me -->
+    <section id="about">
+        <h1 id="about-me">About me</h1>
+
+        <p id="about-description">Hello I'm a software developer! I can help<br>
+             you build a product , feature or website<br>
+              Look through some of my work and<br>
+               experience! If you like what you see and<br>
+                have a project you need coded, don't<br>
+                 hestiate to contact me.</p>
+        <button id="get-my-resume"><p id="get-my-resume-text">Get My Resume</p></button>
+        <img src="img/about-background.svg" alt="" id="about-background">
+    </section>
+
+    <!-- horizontal-line(sections divider) -->
+    <div id="horizontal-rule"></div>
+
+    <!-- skill -->
+    <section id="skills">
+        <!-- frame35 -->
+        <div id="language">
+            <!-- frame30 -->
+            <section id="language-title">
+                <div id="language-icon"></div>
+                <h3 id="language-text">Language</h3>
+            </section>
+
+            <!-- frame29 -->
+            <section id="language-items">
+                <div id="language-items-container">
+                    <p id="language-items-text">Javascript</p>
+                </div>
+                
+                <div id="language-items-container">
+                    <p id="language-items-text">Ruby</p>
+                </div>
+
+                <div id="language-items-container">
+                    <p id="language-items-text">Html</p>
+                </div>
+
+                <div id="language-items-container">
+                    <p id="language-items-text">Css</p>
+                </div>
+
+            </section>
```

Comment:
- [ ] Kindly ensure that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li` which is semantic as `<ul>` is used to represent a list 👍🏽.